### ISSUE TITLE
multiple cryosparcm calls when opening a project

### ISSUE THREAD

**Issue by azazellochg** [scipion-em/scipion-em-cryosparc2#57]
With the profiler I have discovered that cryosparcm is called multiple times when opening any (nothing running) project containing cryosparc2 protocols. The problem is in defineParams for all protocols that contain addComputeSectionParams that has getCryosparcEnvInformation() inside. I understand that cryosparcm cli is called here to detect the version. However there's a cryosparc2_master/version text file that could be used instead and only once (preferably in the plugin init).

This was reported by Dmitry Semchonok:
```
Dear colleagues,
We have a slowing down of any of our projects that contains any of cryoSPARC protocol inside the SCIPION even when no active process is running.
(I do not mean the conversion step).
At the same time, running the same task separately on the cryoSPARC server only does not give any delay in response and finished much faster.
Do you perhaps know the reason and the possible fix?
```

**Comment by azazellochg** [comment]
if i comment:
` versionAllowGPUs = True #parse_version(getCryosparcEnvInformation()) >= parse_version(V2_13_0)`
the project window opens twice faster

**Comment by azazellochg** [comment]
Fixed in 1e38c81
